item-spec/common-metadata.md

```diff
@@ -13,14 +13,9 @@ or [Collection Asset](../collection-spec/collection-spec.md#asset-object).
     - [Relation types](#relation-types)
   - [Provider](#provider)
     - [Provider Object](#provider-object)
-      - [roles](#roles)
   - [Instrument](#instrument)
-    - [Additional Field Information](#additional-field-information)
-      - [platform](#platform)
-      - [instruments](#instruments)
-      - [constellation](#constellation)
-      - [mission](#mission)
-      - [gsd](#gsd)
+  - [Bands](#bands)
```

Comment:
I still do not understand why bands is part of the common metadata and not only an child object for asset

Reply:
So that it can be used in all places.

Reply:
all right then I did not understand the outcome here: https://github.com/stac-extensions/raster/issues/22#issuecomment-1739782633
I thought dehydration was up to implementation and thus did not need the band at item level.

Reply:
In APIs, yes. But you still have static STACs and/or cases where you explicitly want it in Item Properties. That was my understanding.